Internal Server Error on GET /oobi/ (trailing slash)

# Steps to reproduce

Start agent

```
keria start
```

Send a HTTP GET to message endpoint /oobi/

```
curl -v http://localhost:3902/oobi/
```

# Actual result

Keria responds with internal server error

```
*   Trying 127.0.0.1:3902...
* Connected to localhost (127.0.0.1) port 3902 (#0)
> GET /oobi/ HTTP/1.1
> Host: localhost:3902
> User-Agent: curl/7.81.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 500 Internal Server Error
< Content-Type: application/json
< Vary: Accept
< Content-Length: 38
< Server: Ioflo WSGI Server
< Date: Tue, 14 Nov 2023 12:59:46 GMT
<
* Connection #0 to host localhost left intact
{"title": "500 Internal Server Error"}
```

Logs:

```
2023-11-14 13:59:46 [FALCON] [ERROR] GET /oobi/ => Traceback (most recent call last):
  File "/home/lenkan/code/keria/.venv/lib/python3.10/site-packages/keri/db/dbing.py", line 458, in getVal
    return(txn.get(key))
lmdb.BadValsizeError: mdb_get: MDB_BAD_VALSIZE: Unsupported size of key/DB name/data, or wrong DUPFIXED size

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "falcon/app.py", line 365, in falcon.app.App.__call__
  File "/home/lenkan/code/keria/src/keria/end/ending.py", line 57, in on_get
    agent = self.agency.lookup(pre=aid)
  File "/home/lenkan/code/keria/src/keria/app/agenting.py", line 253, in lookup
    if (prefixer := self.adb.aids.get(keys=(pre,))) is not None:
  File "/home/lenkan/code/keria/.venv/lib/python3.10/site-packages/keri/db/subing.py", line 258, in get
    val = self.db.getVal(db=self.sdb, key=self._tokey(keys))
  File "/home/lenkan/code/keria/.venv/lib/python3.10/site-packages/keri/db/dbing.py", line 460, in getVal
    raise KeyError(f"Key: `{key}` is either empty, too big (for lmdb),"
KeyError: "Key: `b''` is either empty, too big (for lmdb), or wrong DUPFIXED size. ref) lmdb.BadValsizeError"
```

# Expected result

Keria returns with HTTP 4xx. Probably with a 404 as with GET /oobi/missing-aid endpoint:

```
$ curl -v http://localhost:3902/oobi/missing-aid
*   Trying 127.0.0.1:3902...
* Connected to localhost (127.0.0.1) port 3902 (#0)
> GET /oobi/abc HTTP/1.1
> Host: localhost:3902
> User-Agent: curl/7.81.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< Content-Type: application/json
< Vary: Accept
< Content-Length: 72
< Server: Ioflo WSGI Server
< Date: Tue, 14 Nov 2023 13:02:36 GMT
<
* Connection #0 to host localhost left intact
{"title": "404 Not Found", "description": "AID not found for this OOBI"}
```

Or a 400 saying that an empty AID is invalid.

--- a/src/keria/app/aiding.py
+++ b/src/keria/app/aiding.py
@@ -442,6 +442,9 @@ class IdentifierResourceEnd:
             name (str): human readable name for Hab to GET
 
         """
+        if not name:
+            raise falcon.HTTPBadRequest(description="name is required")
+
         agent = req.context.agent
         hab = agent.hby.habByName(name)
         if hab is None:
@@ -630,6 +633,9 @@ class IdentifierOOBICollectionEnd:
 
         """
         agent = req.context.agent
+        if not name:
+            raise falcon.HTTPBadRequest(description="name is required")
+
         hab = agent.hby.habByName(name)
         if not hab:
             raise falcon.HTTPNotFound(description="invalid alias {name}")

--- a/src/keria/end/ending.py
+++ b/src/keria/end/ending.py
@@ -48,7 +48,7 @@ class OOBIEnd:
             eid: qb64 identifier prefix of participant in role
 
         """
-        if aid is None:
+        if not aid:
             if self.default is None:
                 raise falcon.HTTPNotFound(description="no blind oobi for this node")
 

--- a/tests/app/test_aiding.py
+++ b/tests/app/test_aiding.py
@@ -315,6 +315,10 @@ def test_identifier_collection_end(helpers):
         res = client.simulate_post(path="/identifiers", body=json.dumps(body))
         assert res.status_code == 202
 
+        res = client.simulate_get(path="/identifiers/")
+        assert res.status_code == 400
+        assert res.json == {'description': 'name is required', 'title': '400 Bad Request'}
+
         res = client.simulate_get(path="/identifiers")
         assert res.status_code == 200
         assert len(res.json) == 2
@@ -1227,6 +1231,11 @@ def test_oobi_ends(helpers):
         op = helpers.createAid(client, "pal", salt)
         iserder = serdering.SerderKERI(sad=op["response"])
         assert iserder.pre == "EHgwVwQT15OJvilVvW57HE4w0-GPs_Stj2OFoAHZSysY"
+
+        # Test empty
+        res = client.simulate_get("/identifiers//oobis?role=agent")
+        assert res.status_code == 400
+        assert res.json == {'description': 'name is required', 'title': '400 Bad Request'}
 
         # Test before endroles are added
         res = client.simulate_get("/identifiers/pal/oobis?role=agent")

--- a/tests/end/test_ending.py
+++ b/tests/end/test_ending.py
@@ -61,6 +61,10 @@ def test_oobi_end(helpers):
         assert res.status_code == 404
         assert res.json == {'description': 'no blind oobi for this node', 'title': '404 Not Found'}
 
+        res = client.simulate_get(path=f"/oobi/")
+        assert res.status_code == 404
+        assert res.json == {'description': 'no blind oobi for this node', 'title': '404 Not Found'}
+
         # Use a bad AID
         res = client.simulate_get(path=f"/oobi/EHXXXXXT15OJvilVvW57HE4w0-GPs_Stj2OFoAHZSys")
         assert res.status_code == 404

